apps/frontend-control/src/lib/apollo.ts

```diff
@@ -8,8 +8,11 @@ import {
   split,
 } from '@apollo/client'
```

Comment:
all three apollo.ts are the same file -> extract this to shared-components at some point